zproject/settings.py

```diff
@@ -329,6 +329,22 @@ def get_secret(key):
     else:
         DATABASES['default']['OPTIONS']['sslmode'] = 'verify-full'
 
+if 'postgres' in DATABASES['default']['ENGINE']:
+    # Set True if you want to support full text search against all languages
+    # USING_PGROONGA = True
+    USING_PGROONGA = False
+else:
+    USING_PGROONGA = False
```

Comment:
> This user-configurable setting for `USING_PGROONGA` should be added to zproject/local_settings_template.py (installed as /etc/zulip/settings.py) so users don't need to patch Zulip to set this setting. You should add its default value of False to DEFAULT_SETTINGS in this file.

Done.




> You should add some docs in a new file docs/full-text-search.md explaining how to setup pgroonga, and linking to the upstream broader docs on pgroonga. (We should probably also have some text in there explaining how the default Zulip full-text search works, but I'm happy to write that).

I've added it with some TODO marks. Could you resolve TODOs for full-text search feature summary and the default full-text search feature description after this pull request is merged?

> Also, the section for this that you add in zproject/local_settings_template.py should have comments linking to that documentation.

Done.